common/common.go

```diff
@@ -425,8 +428,10 @@ var PermissionDeniedAPIError = status.Error(codes.PermissionDenied, "permission
 
 // Redis password consts
 const (
-	DefaultRedisInitialPasswordSecretName = "argocd-redis"
-	DefaultRedisInitialPasswordKey        = "auth"
+	// RedisInitialCredentials is the name for the argocd kubernetes secret which will have the redis password
+	RedisInitialCredentials = "argocd-redis"
+	// RedisInitialCredentialsKey is the key for the argocd kubernetes secret that maps to the redis password
+	RedisInitialCredentialsKey = "auth"
```

Comment:
I'm removing the `Default` prefix. `Default` implies that one can use another value, but there do not appear to be any provisions for this.

If I'm misunderstanding, then someone will tell me and I'll try to improve the comment to explain how one could use a different value / how this is unrelated to those flows.